politeiad/backendv2/tstorebe/inventory.go

```diff
@@ -0,0 +1,428 @@
+// Copyright (c) 2020-2021 The Decred developers
```

Comment:
For all caches, if a cache update fails I consider it terminal condition since the cached data is now incoherent and the only way to fix it is to pull the plug on politeiad and rebuild the cache.  This will allow us to hook panics up to an email or some type of notification that alerts the sysadmin that the cache needs to be rebuilt.

Per our matrix discussion, we've decided that this is acceptable for the time being.  Once we move the cache to reddis/memcached a lot of these panics will go away.

Reply:
tracked in https://github.com/decred/politeia/issues/1369